Update version: OpenRGB.OpenRGB version 0.9.1244

Mystyle-48 opened the pull request:
### Pull request has been created with [komac](https://github.com/russellbanks/Komac) v2.11.2 :rocket:

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/261668)

Mystyle-48 committed:
Update version: OpenRGB.OpenRGB version 0.9.1244

Mystyle-48 committed:
Update OpenRGB.OpenRGB.installer.yaml

stephengillie commented:
Automatic Validation ended with:
>  Installation failed with exit code -1978335226 2025-05-29 16:07:19.774 [CLI ] ShellExecute installer failed: 1603 MSI (s) (80:08) [16:07:19:428]: Product: Soulframe -- Installation failed.
 
 | Hex | Dec | Inverted Dec | Symbol | Description | 
 | --- | --- | --- | --- | --- | 
 | 00000643 | 1603 | -4294965693 | ERROR_INSTALL_FAILURE | Fatal error during installation. OR Previous MSI installer is missing (Or it might mean Install successful but system restart may be required.) Should this be add as an `InstallerSuccessCode` or similar? | 
 | 8A150006 | -1978335226 | 2316632070 | APPINSTALLER_CLI_ERROR_SHELLEXEC_INSTALL_FAILED | Running ShellExecute failed | 


(Automated response - build 1058.)

Mystyle-48 committed:
Update OpenRGB.OpenRGB.installer.yaml

Mystyle-48 committed:
Remove 32-bit version of OpenRGB 0.9.1244

Unfortunately, the 32-bit build is triggering false positives during the validation process.

Mystyle-48 committed:
Update OpenRGB.OpenRGB.locale.en-US.yaml

Mystyle-48 committed:
Update OpenRGB.OpenRGB.installer.yaml

stephengillie commented:
Automatic Validation ended with:
>  Executable C:\Program Files\OpenRGB\OpenRGB.exe returned exit code: -1073741515 Executable C:\Users\Validator\AppData\Local\Microsoft\WinGet\Links\OpenRGB.exe returned exit code: -1073741515
 
 | Hex | Dec | Inverted Dec | Symbol | Description | 
 | --- | --- | --- | --- | --- | 
 | C0000135 | -1073741515 | 3221225781 | STATUS_DLL_NOT_FOUND | Possibly due to a missing dependency. (The most common dependency is `Microsoft.VCRedist.2015+.x64` / `.x86`) | 


(Automated response - build 1059.)

Mystyle-48 committed:
Update OpenRGB.OpenRGB.installer.yaml

Mystyle-48 committed:
Update OpenRGB.OpenRGB.locale.en-US.yaml

Mystyle-48 committed:
Update OpenRGB.OpenRGB.locale.en-US.yaml

Mystyle-48 commented:
Huh. I wonder why validation keeps failing...? The Azure logs don't detail anything in particular.

ItzLevvie commented:
@wingetbot run

ItzLevvie commented:
@wingetbot run

ItzLevvie commented:
@wingetbot run

ItzLevvie commented:
@wingetbot run

ItzLevvie commented:
@wingetbot run

ItzLevvie commented:
@wingetbot run

Mystyle-48 committed:
Update OpenRGB.OpenRGB.locale.en-US.yaml

ItzLevvie commented:
[Policy] Area-External

Mystyle-48 committed:
Update OpenRGB.OpenRGB.locale.en-US.yaml

Mystyle-48 committed:
Update OpenRGB.OpenRGB.locale.en-US.yaml

ItzLevvie commented:
[Policy] Blocking-Issue

ItzLevvie commented:
Close with reason: Stale pull request;
